boards/arm/stm32_min_dev/stm32_min_dev.dtsi

```diff
@@ -70,7 +70,3 @@
 		status = "ok";
 	};
 };
-
-&usb {
-	status = "ok";
-};
```

Comment:
Both black and blue variant support USB device, Is there any reason to remove this ?